bcgov/onroutebc
# ORV2-863-1: Render fields with application data from backend

# Description

This PR improved the Edit Application page by rendering fields with application data got from the backend API.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?
Test steps:

1. Go the the application in progress page by clicking "Permits" tab after logged in to the frontend URL.
2. Click on the hyper link on the application and you will be navigated to the Edit Application page.
3. Expect the fields will be rendered with the application data.
4. Known issue - "Choose From" and "Select Vehicle" can't be rendered since there is no way to determine what is the options that the user saved the application in the past.
5. Known issue - "Vehicle Sub Type" is not rendered when refresh the page.



## Checklist






## Further comments




---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend - Vehicles](https://onroutebc-433-backend-vehicles.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://onroutebc-433-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge-main.yml)
